### ISSUE TITLE
Remove support for specifying an array as a message

### ISSUE THREAD

**Issue by sindresorhus** [Qix-/node-error-ex#4]
https://github.com/Qix-/node-error-ex/commit/bf3ba642168ae7b20bb63863c1065d97f22e51d2#diff-3679dd81e69928644563abcacc1e00ddR77

**Comment by Qix-** [comment]
:+1:

**Comment by sindresorhus** [comment]
@Qix- ping

I was thinking something like https://github.com/Qix-/node-error-ex/commit/49c5006fd8a3cb904a2d615e62ae6cbd5a65058e. I was going to do a PR, but too much weirdness going on and I don't have time to properly look into it. `message` should either be a function or string, or fall back to default.

**Comment by Qix-** [comment]
Yeah I'm not sure what I was trying to achieve with that weird-ass code other than allowing arrays. Go ahead and PR it :+1:
